extensions/2.0/Vendor/EXT_pbr_attributes/Schema/glTF.EXT_pbr_attributes.schema.json

```
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_pbr_attributes glTF extension",
```

Update the schema with newer glTF schema version, ID, and title. Note that the file should also be renamed to `material.EXT_pbr_attributes.schema.json` to indicate that this extends a glTF material.
```suggestion
{
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "$id": "material.EXT_pbr_attributes.schema.json",
    "title": "EXT_pbr_attributes glTF Material Extension",
```